lib/plugins/slack/attachements.js

```diff
@@ -93,10 +93,20 @@ module.exports = {
       let color = 'danger';
 
       const limitMetric = metrics[options.limitMetric];
-      if (limitMetric) {
+      if (limitMetric && options.limitMetric === 'coach') {
```

Comment:
Also the default breakpoints for coachScore are inverted, error should be lower than warning.